ksqldb-cli/src/test/java/io/confluent/ksql/cli/console/ConsoleTest.java

```diff
@@ -393,22 +394,28 @@ public void shouldPrintExplainQueryWithError() {
     final QueryDescriptionEntity queryEntity = new QueryDescriptionEntity(
         "statement",
         new QueryDescription(
-            new QueryId("id"),
-            "statement",
-            Optional.empty(),
-            ImmutableList.of(
-                new FieldInfo(
-                    "name",
-                    new SchemaInfo(SqlBaseType.STRING, ImmutableList.of(), null),
-                    Optional.empty())),
-            ImmutableSet.of("source"),
-            ImmutableSet.of("sink"),
-            "topology",
-            "executionPlan",
-            ImmutableMap.of("overridden.prop", 42),
-            ImmutableMap.of(new KsqlHostInfoEntity("foo", 123), KsqlQueryStatus.ERROR),
-            KsqlQueryType.PERSISTENT,
-            ImmutableList.of(new QueryError(timestamp, "error", Type.SYSTEM))
+          new QueryId("id"),
+          "statement",
+          Optional.empty(),
+          ImmutableList.of(
+              new FieldInfo(
+                  "name",
+                  new SchemaInfo(SqlBaseType.STRING, ImmutableList.of(), null),
+                  Optional.empty())),
+          ImmutableSet.of("source"),
+          ImmutableSet.of("sink"),
+          "topology",
+          "executionPlan",
+          ImmutableMap.of("overridden.prop", 42),
+          ImmutableMap.of(new KsqlHostInfoEntity("foo", 123), KsqlQueryStatus.ERROR),
+          KsqlQueryType.PERSISTENT,
+          ImmutableList.of(new QueryError(timestamp, "error", Type.SYSTEM)),
+          ImmutableSet.of(
```

Comment:
The changes have not been merged into streams yet to use the new TaskMetadata. Also I have not updated the condition for this test so it will fail rn

Reply:
Could you rebase and update this PR?

Reply:
I don't think the sync job has finished. The changes are not showing up yet even after a rebase  and clean install

Reply:
Cool, sounds good, so should we proceed with the PR or wait for the sync to be done?

Reply:
We can proceed, the only changes after the sync is that there will be a few fields added to `StreamsTaskMetadata` but they will only change the json out put. Everything else will be the same.